tests/do.sh

```diff
@@ -5,7 +5,7 @@ cd "$(dirname "${0}")"
 READER="../example/ndpiReader -p ../example/protos.txt -c ../example/categories.txt"
 
 RC=0
-PCAPS=`cd pcap; /bin/ls *.pcap`
+PCAPS=`cd pcap; /bin/ls *.pcap*`
```

Comment:
I'm not a fan of using wildcards for filename suffixes as this matches literally anything with a `*.pcap* ` infix.

My recommendation would be `/bin/ls *.pcap *.pcapng`.

Reply:
@lnslbrty Thanks! I fixed it.